### PR TITLE
Add About Page

### LINKED ISSUE #72: Add Project Website (About Page).

### PR DESCRIPTION
**Pull request by charkour**
This PR adds the About Page according to the [specifications on the CS 396 website](https://cs.calvin.edu/courses/cs/396/website.html). 

*To Test*
1. `npm i` and `npm start`
1. I recommend opening up the link to the CS website and look at that while you read through the About Page. Please check the grammar and please let me know if you want anything changed! You could even make a "suggested changes" thing if you want. After that looks good, then you can look at the code to make sure that looks all good. Thanks for the reviews.

*Components added*
- the about page
- text sections for the about page
- animated titles
- animated imaged you can mouse over
- a footer

This was a bigger PR than I was hoping for. So I apologize for that. 

When this is merged, then we can add a link to the About Page to the class Moodle.

closes #72

### CODE REVIEW

File: client-course-schedulizer/src/utilities/helpers/teamMembers.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    bio:
+      "Computer Science student at Calvin University. Charles plans to start his software career post-graduation with Tekton as an ecommerce developer.",
+    name: "Charles Kornoelje",
+    photo:
+      "https://avatars2.githubusercontent.com/u/33156025?s=460&u=fb61bdf55f17108f9687c334b4b4abc4b09c7259&v=4",
+    website: "https://github.com/charkour",
+  },
+  {
+    bio:
+      "Computer Science Professor Department Chair at Calvin University. Keith's research interests are in Natural Language Engineering and Human-Computer Interaction.",
```
Comment (Syobnaf) [inline client-course-schedulizer/src/utilities/helpers/teamMembers.ts:36]:
```suggestion
      "Computer Science Professor and Department Chair at Calvin University. Keith's research interests are in Natural Language Engineering and Human-Computer Interaction.",
```

File: client-course-schedulizer/src/utilities/helpers/teamMembers.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    name: "Professor Keith VanderLinden",
+    photo:
+      "https://avatars0.githubusercontent.com/u/4930536?s=460&u=216f27d175496fec82a87e0c1c1c1514f73997da&v=4",
+    website: "https://calvin.edu/directory/people/keith-vander-linden",
+  },
+  {
+    bio:
+      "Mathematics and Statistics Professor and Department Chair at Calvin University. Randall's interests are in computational statistics, scientific computing, and statistics education.",
```
Comment (bryantgeorge) [inline client-course-schedulizer/src/utilities/helpers/teamMembers.ts]:
```suggestion
      "Mathematics and Statistics Professor and Department Chair at Calvin University. Randall's research interests are in Computational Statistics, Scientific Computing, and Statistics Education.",
```

File: client-course-schedulizer/src/utilities/helpers/teamMembers.ts (hunk 1 above)
Comment (bryantgeorge) [inline client-course-schedulizer/src/utilities/helpers/teamMembers.ts]:
```suggestion
      "Computer Science Professor and Department Chair at Calvin University. Keith's research interests are in Natural Language Engineering and Human-Computer Interaction.",
```

Comment (bryantgeorge) [inline client-course-schedulizer/src/utilities/helpers/teamMembers.ts]: Should we refer to Profs as Keith and Randall or VanderLinden and Pruim?

File: client-course-schedulizer/src/utilities/helpers/teamMembers.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    bio:
+      "Computer Science student at Calvin University. Jonathan is a hard worker, a faster runner, and has previous web development experience.",
+    name: "Jonathan Ellis",
+    photo:
+      "https://avatars3.githubusercontent.com/u/49655167?s=460&u=b81f525db7da58b19bc28588282b366cd8b748ab&v=4",
+    website: "https://github.com/Jonri2",
+  },
+  {
+    bio:
+      "Computer Science and Mathematics at Calvin University. Bryant hoping to go into software development while keeping Mathematics as a lifelong hobby.",
```
Comment (bryantgeorge) [inline client-course-schedulizer/src/utilities/helpers/teamMembers.ts]:
```suggestion
      "Computer Science and Mathematics Student at Calvin University. Bryant is hoping to go into software development while keeping Mathematics as a lifelong hobby.",
```

File: client-course-schedulizer/src/utilities/helpers/teamMembers.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  name: string;
+  photo: string;
+  website: string;
+}
+
+/* Basic information about the project team members */
+export const team: TeamMember[] = [
+  {
+    bio:
+      "Computer Science student at Calvin University. Jonathan is a hard worker, a faster runner, and has previous web development experience.",
```
Comment (bryantgeorge) [inline client-course-schedulizer/src/utilities/helpers/teamMembers.ts]: Capitalizing student to remain more consistent with capitalizing professor and department chair (but if you disagree that's fine too)
```suggestion
      "Computer Science Student at Calvin University. Jonathan is a hard worker, a faster runner, and has previous web development experience.",
```

File: client-course-schedulizer/src/utilities/helpers/teamMembers.ts [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    bio:
+      "Computer Science and Mathematics at Calvin University. Bryant hoping to go into software development while keeping Mathematics as a lifelong hobby.",
+    name: "Bryant George",
+    photo:
+      "https://avatars1.githubusercontent.com/u/51130302?s=460&u=83737db9aeff2b377654a3e0a7cd3dc39f54f7ff&v=4",
+    website: "https://github.com/Syobnaf",
+  },
+  {
+    bio:
+      "Computer Science student at Calvin University. Charles plans to start his software career post-graduation with Tekton as an ecommerce developer.",
```
Comment (bryantgeorge) [inline client-course-schedulizer/src/utilities/helpers/teamMembers.ts]:
```suggestion
      "Computer Science Student at Calvin University. Charles plans to start his software career post-graduation with Tekton as an ecommerce developer.",
```

**Review by bryantgeorge — CHANGES_REQUESTED** [review]
Looks really nice (especially the animation and linkage for the team images)! Just a few grammar/consistency things to address, otherwise LGTM (probably fine to merge after addressing the suggested changes). Also remember to use https://senior-knights.github.io/about as the link (instead of the base app link) for the class Moodle.

File: client-course-schedulizer/src/utilities/helpers/teamMembers.ts [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+      "https://avatars3.githubusercontent.com/u/49655167?s=460&u=b81f525db7da58b19bc28588282b366cd8b748ab&v=4",
+    website: "https://github.com/Jonri2",
+  },
+  {
+    bio:
+      "Computer Science and Mathematics at Calvin University. Bryant hoping to go into software development while keeping Mathematics as a lifelong hobby.",
+    name: "Bryant George",
+    photo:
+      "https://avatars1.githubusercontent.com/u/51130302?s=460&u=83737db9aeff2b377654a3e0a7cd3dc39f54f7ff&v=4",
+    website: "https://github.com/Syobnaf",
```
Comment (bryantgeorge) [inline client-course-schedulizer/src/utilities/helpers/teamMembers.ts]: I've recently changed my GitHub username:
```suggestion
    website: "https://github.com/bryantgeorge",
```

File: client-course-schedulizer/src/utilities/helpers/teamMembers.ts (hunk 1 above)
Comment (charkour) [reply client-course-schedulizer/src/utilities/helpers/teamMembers.ts]: Yeah, that sounds good. Thanks!

**Comment by charkour** [comment]
Thanks for the fixes! Can't believe I missed those.

**Pull request merged**